### DISCUSSION TITLE
Allow access to and transfer of all your "xxxxxxxx"? issue

### DISCUSSION THREAD

**Discussion by white-beareth** [Synthetixio/synpress#730] [Q&A] [answered]
With the latest MM update I find that the permissions are no longer one-step.

I'm using cy.confirmMetamaskPermissionToSpend(), it confirms the transaction, but being a Permission by ALL NFT, it does not show the amount input, but directly shows the Approve or Reject screen, but I have no way to approve that step. 

![image](https://user-images.githubusercontent.com/128830623/234381742-5c6b10fe-f8fd-4fa8-b246-b3143ddf4497.png)

Can someone help me to solve how we do with screens that are typical in NFT tests where we must give permission to a complete collection?

**_You can reproduce it using MetaMask test-dapp as follows:_**
1-Enter Test-Dapp MetaMask
2-Connect the MM
3-In the Collectibles section, click on Deploy
4-In the Collectibles section click on Mint
5-In the Collectibles section click on SET APPROVAL FOR ALL
6-There will open the MM to Confirm and then Approve All and there is where I can not move forward

**Comment by TorresJJ** [comment]
Getting the same error here! 
![image](https://user-images.githubusercontent.com/121894848/234569541-89f7c163-201f-401c-943b-9c43dce94400.png)

**Comment by TorresJJ** [comment]
@drptbl I have created a pull request that solves this issue. Pleas review and let me know if it all looks good.

https://github.com/Synthetixio/synpress/pull/732

**Comment by r3kt-eth** [comment, marked as answer]
This should be fixed with latest release of synpress.
